libutils/pcre_wrap.c

```diff
@@ -0,0 +1,495 @@
+/*
+
+ * The original pcrs code came from ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/Contrib/pcrs-0.0.3-src.tar.gz
+
+ * This is a rewrite of PCRS with the same API and behavior.
+
+ */
+
+
+
+#include <string.h>
+#include <ctype.h>
```

Comment:
Always include our `<platform.h>` on top for portability reasons, which BTW makes the standard C library includes here redundant, so you can remove them.